recipes/simdna/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "simdna" %}
+{% set version = "0.4.2" %}
+{% set sha256 = "0b16876b90a8628afc145a3b197017b134374da479e20d72c63fda2a30102c09" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - matplotlib
+    - scipy
+    - numpy >=1.9
+
+test:
+  imports:
+    - simdna
+
+about:
+  home: https://github.com/kundajelab/simdna
+  license: MIT
```

Comment:
MIT license requires a `license_file` too. You should file an issue about it not being in the distribution, or just send a PR to fix it (it's easy; [example](https://github.com/Ceasar/easywatch/pull/10)), add a comment here pointing to that, and in the meantime [package the license manually](https://conda-forge.org/docs/meta.html#packaging-the-license-manually).